src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/DataSetDataTableInWebSerializableObjectGraphTests.cs

```diff
@@ -90,7 +90,9 @@ public class MyClass
 
         private static async Task VerifyServiceModelCSharpAsync(string source, params DiagnosticResult[] expected)
         {
+#pragma warning disable CA5386 // Avoid hardcoding SecurityProtocolType value
```

Comment:
Might be worth updating the analyzer to not report on test method, what do you think?

Reply:
I think it is unlikely that we see this violation in test code except the test project in this repo. Not sure if it is worth the effort.